.travis-mirage.sh

```diff
@@ -16,19 +16,20 @@ TMP_BUILD=$(mktemp -d)
 cd ${TMP_BUILD}
 
 get .travis-ocaml.sh
+get src/yorick/yorick.mli
+get src/yorick/yorick.ml
+get src/travis_mirage.ml
+
+exit 1
```

Comment:
Should this `exit` be here? If so, what's the rest of the file for?

Reply:
whoops ...